### PR TITLE
Bug 1215754 - Make AdvancedTelemetry transmit properly after airplane…

### PR DESCRIPTION
**Pull request by tamarahills**
… mode turned off.

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7c1296b3f3783ea9e99df123db42379f37ec6e4a

File: apps/system/js/advanced_telemetry.js [hunk 1]
```diff
@@ -379,13 +377,17 @@
...
-      });
+      if (navigator.onLine) {
+        request.send({
+          timeout: AT.REPORT_TIMEOUT,
+          onload: onload,
+          onerror: retry,
+          onabort: retry,
+          ontimeout: retry
+        });
+      } else {
```
Comment (russnicoletti) [inline apps/system/js/advanced_telemetry.js]: The changes look good. I can see how they solve the problem, and the code is much cleaner. My only constructive comment is the readability would be improved by checking 'navigator.onLine' at the beginning of the 'send' function. (Jan's 'bailout' style is rubbing off on me).

Comment (tamarahills) [inline apps/system/js/advanced_telemetry.js]: Makes sense.  I will add this.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e57c992e2521b5ffe34614664f40c82ae1f22752

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4fac69493d83593b92a316bc7f0e887436c3781a

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=28a6c78cf6a4ca2e6b74e2c823a13ae7d65256cd

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c784c86d01097cec45e13b825fad9cbe6b80e59f

**Pull request merged**